packages/react-network/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@shopify/react-network",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "A collection of components that allow you to set common HTTP headers from within your React application.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
```

Comment:
🔥 the explicit registry (per [the guide](https://development.shopify.io/guides/package_cloud_nodejs/)).